package.json

```diff
@@ -48,15 +45,19 @@
     "optimizecss": "postcss dist/css/style.css -c postcss.config.js -o dist/css/style.css",
     "scss": "node-sass src/scss/style.scss -o dist/css",
     "gutenberg": "node-sass src/scss/gutenberg-blocks-style.scss -o dist/css",
-    "lint": "eslint src/js || true",
-    "lint-scss": "stylelint src/scss/**/*.scss --syntax scss --config .stylelintrc || true",
+    "lint": "npm run lint-js && npm run lint-scss",
+    "lint:js": "npm run lint-js",
+    "lint:scss": "npm run lint-scss",
+    "lint-js": "./node_modules/.bin/eslint src/js || true",
+    "lint-scss": "./node_modules/.bin/stylelint src/scss/**/*.scss --syntax scss --config .stylelintrc || true",
```

Comment:
Did _not_ install `css-coding-standards` wd_s uses (according to Corey) a more modern setup with stylelint that we should actually be using. Will be stealing this later to put in css-coding-standards and eventually install.